### DISCUSSION TITLE
Wildcard domain question/issue

### DISCUSSION THREAD

**Discussion by jpatronas** [vercel/community#1356] [Help]
I am using vercel wildcard subdomains using an express app and I think I have things setup properly.  I can go to abc.domain.com and abc2.domain.com and it will execute the appropriate route, however when I reference the host from express app I get the deployment URL and not the url entered in the browser.  

For example, when I go to https://abc.domain.com and then inside my route do a req.get(‘host’) I get vercel-project-name-18idt68d8-project.vercel.app instead of abc.domain.com which is what I want.  I'd like to parse the host and get 'abc' but can't seem to get that value.

**Comment by amyegan** [comment]
Hi @jpatronas! I tried this in my own test app and wasn't able to reproduce the issue. Do you have a public repo or minimal reproduction that I can use to get a better look at this problem?

**Comment by jpatronas** [comment]
Yes,I am traveling until Monday.  Can you keep the ticket open and I will contact you.  It’s important for me to figure this out.  You guys are rockstar support team!!Sent from my iPhoneOn Jan 25, 2023, at 12:11 PM, Amy ***@***.***> wrote:﻿
Hi @jpatronas! I tried this in my own test app and wasn't able to reproduce the issue. Do you have a public repo or minimal reproduction that I can use to get a better look at this problem?

**Reply by amyegan** [reply]
Safe travels! We can work on this together when you're back online.

**Comment by jpatronas** [comment]
Thank you!Sent from my iPhoneOn Jan 26, 2023, at 9:09 AM, Amy ***@***.***> wrote:﻿
Safe travels! We can work on this together when you're back online.
